### PR TITLE
[AutoScheduler] Improve warning messages

### PR DESCRIPTION
**Pull request by merrymercy**
Improve the warning messages for #6903

### CODE REVIEW

File: python/tvm/relay/op/strategy/cuda.py [hunk 1]
```diff
@@ -224,7 +231,7 @@ def conv2d_strategy_cuda(attrs, inputs, out_type, target):
             if use_auto_scheduler and judge_winograd_auto_scheduler:
                 strategy.add_implementation(
                     wrap_compute_conv2d(topi.nn.conv2d_winograd_nhwc),
-                    wrap_topi_schedule(tvm.te.create_schedule),
```
Comment (merrymercy) [inline python/tvm/relay/op/strategy/cuda.py:227]: `tvm.te.create_schedule` is not a valid topi schedule. Their signatures are different

File: python/tvm/relay/op/strategy/cuda.py [hunk 2]
```diff
@@ -224,7 +231,7 @@ def conv2d_strategy_cuda(attrs, inputs, out_type, target):
...
+                    naive_schedule,  # this implementation should never be picked by autotvm
```
Comment (merrymercy) [inline python/tvm/relay/op/strategy/cuda.py:234]: `tvm.te.create_schedule` is not a valid topi schedule. Their signatures are different. So I created a new one.

File: python/tvm/relay/op/strategy/cuda.py [hunk 3]
```diff
@@ -101,6 +101,17 @@ def schedule_lrn_cuda(attrs, outs, target):
...
 
+def naive_schedule(_, outs, target):
+    """Return the naive default schedule"""
+    if "gpu" in target.keys:
+        # For GPU, we at least need thread binding to make a valid schedule.
+        # So the naive schedule cannot be compiled.
+        raise RuntimeError(
+            "Cannot compile for GPU targets if no tuned schedule is found. Please see the warning messages above for more information about the failed workloads."
+        )
+    return tvm.te.create_schedule(outs[-1].op)
```
Comment (comaniac) [inline python/tvm/relay/op/strategy/cuda.py:112]: Maybe we need to add a comment saying that it would be a problem for the DAG with multiple outputs.

**Review by comaniac — APPROVED** [review]
LGTM. Just a nit

**Pull request merged**